src/main/resources/config/liquibase/changelog/20221225120000_changelog.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="stefanwaldhauser" id="20221225120000-1">
+        <addColumn tableName="conversation">
+            <column name="discriminator" type="CHAR(1)" defaultValue="O">
+                <constraints nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(20)">
+                <constraints nullable="true" />
+            </column>
+            <column name="description" type="VARCHAR(250)">
+                <constraints nullable="true" />
+            </column>
+            <column name="is_public" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="true" />
+            </column>
+            <column name="topic" type="VARCHAR(250)">
+                <constraints nullable="true" />
+            </column>
+            <column name="creator_id" type="BIGINT">
+                <constraints nullable="true" />
+            </column>
+            <column name="is_archived" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="true" />
+            </column>
+            <column name="is_announcement" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="true" />
+            </column>
+        </addColumn>
+        <addColumn tableName="conversation_participant">
+            <column name="is_moderator" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="true" />
+            </column>
+            <column name="is_hidden" type="boolean" defaultValueBoolean="false">
```

Comment:
I don't quite understand this here... You are creating a ConversationParticipant for every user in a conversation.
How do you track the hidden/favorite flag per user then? Wouldn't that require an n-to-n relation?

Reply:
```
                     Contains user specific settings for the conversation
                            │
                            │
                            │
                            │
                            │
     1         n            ▼            n      1
User   ────────  ConversationParticipant  ──────  Conversation
```

Reply:
I do not like the name participant to be fair. I think "Subscription" would be better, so basically you can use this entity to store user specific settings per conversation.

Reply:
Added uniqueness constraint in eab79f17261a0b27078cbc9ac54d22f552dcd1ce to enforce this

Reply:
Okay, I think my misunderstanding was that the "hidden" and "favorite" flags were for the user, when actually they belong to the conversation.

Reply:
I think you already have such a constraint? https://github.com/ls1intum/Artemis/pull/5822/files/7915be8619ff2bbfa5c9eeab196c0713ee625901#diff-f9d1bc6e238168a0ff3a44a1a08d541ecf1c0adccda8a490a1b5853b22e17635R42

Reply:
Yeah removed it again in the latest commit :)